microutil/preprocess.py

```diff
@@ -110,40 +114,45 @@ def xarray_plls(log_power_spec, logR):
     )
 
 
-def squash_zstack(data, squash_fn="max", bf_name="BF", channel_name="channel", transpose=True):
+def squash_zstack(
+    data,
+    squash_fn="max",
```

Comment:
```suggestion
    squash_fn="mean",
```

nows as good a time as ever to swap the default?

Reply:
Also probs a good time to add a real docstring

Reply:
Great call swapping the default i meant to and forgot.